server/src/main/java/com/showmetheroapes/paracord/models/domain/StrandModel.java

```diff
@@ -0,0 +1,28 @@
+package com.showmetheroapes.paracord.models.domain;
+
+import lombok.*;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "strands")
+@NoArgsConstructor
+@Setter
+@Getter
```

Comment:
Oh interesting, so these annotations will provide getting and setters for the properties then? That's nifty